### DISCUSSION TITLE
Is there a way to specify the custom filename format when adding new languages?

### DISCUSSION THREAD

**Discussion by Wohlstand** [WeblateOrg/weblate#9622] [Q&A] [answered]
Hello!

Recently I got a small question, is possible to give an alternative format for the "language-locale" string?

For example, by default, the thing produces the `xx` and `xx_YY` format. In a case of my application I use the format `xx` and `xx-yy`.

I saw that WebLate made the new file using the built-in `xx_YY` format. Is possible to specify the custom format for filenames?

**Comment by nijel** [comment, marked as answer]
Not fully custom, but you can choose from many, see https://docs.weblate.org/en/latest/admin/projects.html#component-language-code-style

**Reply by Wohlstand** [reply]
Oh, I see, my format is actually `BCP style using hyphen as a separator, lower cased`. Just now I set it, now all the fine. Thanks for the hint!
